frontend/src/pages/AccountSend.vue

```diff
@@ -325,11 +572,21 @@ function useSendForm() {
   const attemptedNetworkChange = ref(false);
   const connectRedirectTo = ref('send');
 
+  // Batch Send Form Parameters
+  const Sends = ref<SendData[]>([]);
```

Comment:
variables should be lower case
```suggestion
  const batchSends = ref<SendData[]>([]);
```

Reply:
Thanks for the reminder. Renamed it here: https://github.com/ScopeLift/umbra-protocol/pull/491/commits/7c4e67a7730b4a83fbcb6dd87f3ad74cafbce2ad